src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -0,0 +1,93 @@
+export type ColumnValueType = 'numeric' | 'string' | 'unknown';
+
+// todo(leedongwei): Find the correct types
+export const AGGREGATIONS = {
+  latest_event: {
+    type: 'string',
+    isSortable: true,
+  },
+  last_seen: {
```

Comment:
This and `latest_event` aren't aggregate functions as they don't take parameters. They are column aliases, that wrap up an aggregation though.